src/views/Onboarding/OnboardingSetup.vue

```diff
@@ -44,7 +44,7 @@
       v-show="currentStep === 'password'"
       :imported="imported"
       @on-unlock="onUnlock"
-      @currentStep="currentStep = 'confirm'"
+      @currentStep="changeCurrentStep"
```

Comment:
I think this event should be renamed to "on-cancel"
here and in in OnboardingPassword